src/main/kotlin/net/leanix/vsm/gitlab/broker/connector/application/WebhookConsumerService.kt

```diff
@@ -18,7 +18,9 @@ import net.leanix.vsm.gitlab.broker.shared.exception.NamespaceNotMatchException
 import org.springframework.beans.factory.annotation.Value
 import org.springframework.stereotype.Service
 
-val PROJECT_EVENTS = listOf("project_create", "project_update", "project_rename", "project_transfer")
+const val PROJECT_DESTROY_WEBHOOK_EVENT_NAME = "project_destroy"
+val PROJECT_EVENTS =
```

Comment:
I think you can migrate the constants to this class:
https://github.com/leanix/vsm-gitlab-broker/blob/747365891db7036a477455943746950c483883e7/src/main/kotlin/net/leanix/vsm/gitlab/broker/shared/Constants.kt

Reply:
done